frontend/src/features/admin-form/create/end-page/EndPageContent.tsx

```diff
@@ -51,14 +49,11 @@ export const EndPageView = ({ ...props }: FlexProps): JSX.Element => {
       mb={0}
       flex={1}
       bg="neutral.200"
-      // Using margin for margin collapse when there are inline messages above.
```

Comment:
may still need this, this is for when there are webhook messages. do they look weird now? should have a story for it. if it looks fine then ignore this

Reply:
webhook message is only displayed in builder and design tabs, since endpage is now a separate tab, we don't need it anymore haha.